Doc/install/INSTALL.rst

```diff
@@ -19,7 +19,7 @@ GCC is recommende as the C compiler, which is usually available as part of the s
 2) Mac OS X
 ------------
 
-Install Apple’s XCode suite from the App Store, and then from the Xcode options also install the optional command line utilities.
+Install Apple’s XCode suite from the App Store, and then install the commandline tools using command- ``xcode-select --install`` on the terminal.
```

Comment:
You don't need the XCode suite at all anymore. I think the wording on http://biopython.org/wiki/Download is still accurate (I don't have a clean Mac to double check this on, those I have access to already have the command line tools).

Also missing  a space in "command line" and there is an extra minus/hyphen before the the command.

Reply:
I wasn't sure earlier about this. I confirmed this right now on a friend's Mac. Running the command itself is enough for command-line tools and installing XCode also ensures that you get the command-line tools.

Reply:
Thanks for doing that - good to know what we're recommending on http://biopython.org/wiki/Download is still current for getting the compilers on macOS 👍